css/src/components/tabs.scss

```diff
@@ -0,0 +1,110 @@
+$tabs-padding: 0.5em 1.25em !default;
+$tabs-padding-large: 1em 1.5em !default;
+
+$tab-font-color: $primary !default;
+$tabs-hover-background-color: $primary-background !default;
+$tabs-hover-border-color: $primary-hover !default;
+$tabs-active-hover-background-color: $primary-hover !default;
+$tabs-active-hover-border-color: $primary-hover !default;
+$tabs-active-background-color: $primary-active;
+$tabs-active-border-color: $primary-active !default;
+$tabs-active-color: $text-invert !default;
+
+.tabs {
+	display: grid;
+	justify-content: center;
+	width: 100%;
+	grid-gap: 1em;
+	grid-template-columns: max-content auto max-content;
+	white-space: nowrap;
+
+	.tab-list {
+		grid-column: 2;
+		display: flex;
+		align-items: center;
+		justify-content: flex-start;
+		border-radius: $border-radius-rounded;
+		box-shadow: $box-shadow-medium;
+		overflow-x: hidden;
+	}
+
+	.tab-previous,
+	.tab-next {
+		display: flex;
+		align-items: center;
+		padding: $tabs-padding;
+		border: none;
+		border-radius: $border-radius-rounded;
+		background-color: $tabs-active-background-color;
+		color: $tabs-active-color;
+		font-weight: 500;
+		box-shadow: $box-shadow-medium;
+
+		&:hover {
+			border-color: $tabs-active-hover-border-color;
+			background-color: $tabs-active-hover-background-color;
+		}
+	}
+
+	.tab-control {
+		display: flex;
+		position: relative;
+		align-items: center;
+		justify-content: center;
+		padding: $tabs-padding;
+		border: none;
+		border-radius: $border-radius-rounded;
+		background-color: transparent;
+		color: $tab-font-color;
+		font-weight: 500;
+		cursor: pointer;
+		appearance: none;
+		vertical-align: top;
+
+		&:hover {
+			border-color: $tabs-hover-border-color;
+			background-color: $tabs-hover-background-color;
+			z-index: $zindex-hover;
+		}
+
+		@include tablet {
+			padding: $tabs-padding;
+		}
+	}
+
+	&.tabs-lg {
+		.tab-control,
+		.tab-previous,
+		.tab-next {
+			padding: $tabs-padding-large;
```

Comment:
can we do this size with font-size? that's one reason why we do inner `em` sizing within these components.

Reply:
updated